### PR TITLE
Fixed to handle single quotes properly

### PR DESCRIPTION
**Pull request by gouniLee**
* Fixed an issue where didn't handle single quotes properly.
* sample: https://github.com/iLib-js/ilib-loctool-samples/pull/48

### CODE REVIEW

File: QMLFile.js [hunk 1]
```diff
@@ -52,8 +52,10 @@ QMLFile.unescapeString = function(string) {
         replace(/^\\\\/, "\\").             // unescape backslashes
         replace(/([^\\])\\\\/g, "$1\\").
         replace(/^\\"/, '"').
+        replace(/^\\'/, "'").
         replace(/([^\\])\\"/g, '$1"').
-        replace(/\\"/g, '"');
+        replace(/\\"/g, '"').
+        replace(/\\'/g, "'");
```
Comment (ehoogerbeets) [inline QMLFile.js:58]: This maps \\' to ' twice, no?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 ilib-webos-loctool-qml is a plugin for the loctool allows it to read and localize qml files. This plugins is optimized for webOS platform.
 
 ## Release Notes
+v1.6.0
+* Fixed an issue where didn't handle single quotes properly.
```
Comment (ehoogerbeets) [inline README.md:6]: If it is a bug fix, this should be v1.5.1. If it is a new feature, then it is v1.6.0

File: QMLFile.js (hunk 1 above)
Comment (gouniLee) [reply QMLFile.js:58]: The problem case was `qsTr("Don\'t save")`, loctool converted it as `"Don\\'t save"` so couldn't find the translation properly. 
I don't think it maps \' twice. am I missing some other case?

File: README.md (hunk 2 above)
Comment (gouniLee) [reply README.md:6]: I updated it

File: QMLFile.js (hunk 1 above)
Comment (ehoogerbeets) [reply QMLFile.js:58]: Line 55 and 58 both map the escaped single quote, though line 55 has the "beginning of line" carat in it for some reason. Line 58 should catch both cases

Comment (gouniLee) [reply QMLFile.js]: ah..I update it.

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**